src/app/content/utils.ts

```diff
@@ -137,3 +139,7 @@ export const loadPageContent = async(loader: ReturnType<AppServices['archiveLoad
   const page = await loader.page(pageId).load();
   return page.content;
 };
+
+export const getBookPipelineVersion = (book: Book): string => {
```

Comment:
we have this getter to make sure the latest pipeline is being used: https://github.com/openstax/rex-web/blob/main/src/gateways/createBookConfigLoader.ts#L44

Reply:
Ah okay so it's probably safe then to also make this search hook use it? https://github.com/openstax/rex-web/pull/1636/files#diff-a3aea54c89e3ebfc1fdca9da914d39cf889cc9b3d2de9574a1b207fd243fce6cL31

Reply:
I can't see why not. In the hook I think you'd want to do something like 

```
const archiveUrl = getBookVersionFromUUIDSync(bookId)?.archiveOverride || getArchiveUrl();
const pipeline = archiveUrl.replace(/^\/apps\/archive\//, '');
```

Reply:
Thanks makes sense, for some reason thought it might be handled specially there, but forgot we just added that method recently